deploy/kubernetes/configmap.yaml

```diff
@@ -6,31 +6,79 @@ metadata:
 data:
   ncp.ini: |
 
-    [vc]
 
-    # IpAddress or Hostname of VC
-    #vc_endpoint = <None>
+    [DEFAULT]
+
+    # If set to true, the logging level will be set to DEBUG instead of the
+    # default INFO level.
+    #debug = False
+
+    # If set to true, log output to standard error.
+    #use_stderr = True
+
+    # Destination to send api log to. STDOUT or STDERR for console output. FILE
+    # to write log to file configured in "api_log_file". NONE to disable api
+    # log.
+    # Choices: STDOUT STDERR FILE NONE
+    #api_log_output = NONE
+
+    # Name of log file to send API access log to.
+    #api_log_file = ncp_api_log.txt
+
+    # Interval in seconds to logs api call to output configured in
+    # api_log_output
+    #api_log_interval = 60
+
+    # When api_log_output is not NONE, this option determines if api calls
+    # should be collected per NSX cluster or individual NSX manager.
+    # Choices: API_LOG_PER_ENDPOINT API_LOG_PER_CLUSTER
+    #api_log_mode = API_LOG_PER_ENDPOINT
+
+    # If set to true, use syslog for logging.
+    #use_syslog = False
+
+    # The base directory used for relative log_file paths.
+    #log_dir = <None>
+
+    # Name of log file to send logging output to.
+    #log_file = <None>
+
+    # max MB for each compressed file. Defaults to 100 MB.
+    #log_rotation_file_max_mb = 100
+
+    # max MB for each compressed file for API logs.Defaults to 10 MB.
+    #api_log_rotation_file_max_mb = 10
+
+    # Total number of compressed backup files to store. Defaults to 5.
+    #log_rotation_backup_count = 5
+
+    # Total number of compressed backup files to store API logs. Defaults to 5.
+    #api_log_rotation_backup_count = 5
+
+    # Log level for the root logger. If debug=True, the default root logger
+    # level will be DEBUG regardless of the value of this option. If this
+    # option is unset, the default root logger level will be either DEBUG or
+    # INFO according to the debug option value
+    # Choices: NOTSET DEBUG INFO WARNING ERROR CRITICAL
+    #loglevel = <None>
 
-    # The SSO domain associated with the deployment
-    #sso_domain = vsphere.local
 
-    # VC API server HTTPS port.
-    #https_port = 443
 
 
     [coe]
 
     # Container orchestrator adaptor to plug in.
-    adaptor = kubernetes
+    #adaptor = kubernetes
 
     # Specify cluster for adaptor.
-    cluster = k8scl-one
+    #cluster = k8scluster
```

Comment:
It's better to set `k8scluster` as default value. But do you think if this change will have upgrade regression concern?

Reply:
hmm... in NCP the default value is indeed `k8scluster` for now. So I think if there's no upgrade issue for `cluster = k8scluster`, then there won't be upgrade issue for `#cluster = k8scluster`, do you think it makes sense?

Reply:
make sense.